server/views/partials/searchHeader.njk

```diff
@@ -3,7 +3,7 @@
 <div class="govuk-grid-column-full">
     <div class="block-background">
         <div class="moj-search govuk-!-width-one-half">
-            <form action="/search/" method="POST">
+            <form action="/search/" method="POST" novalidate>
```

Comment:
The `novalidate` attribute is recommended in gov.uk to stop the browser doing any HTML5 validation.